### PR TITLE
CA Amendment

### PR DESCRIPTION
**Pull request by hulto**
Updated terminology

### CODE REVIEW

File: constitution/article-06.tex [hunk 1]
```diff
@@ -22,7 +22,8 @@
     \item President
     \item Secretary
     \item Treasurer
-    \item Competition Architect 
+    \item ISTS Competition Architect 
+    \itme IRSeC Competition Architect
```
Comment (rpg4231) [inline constitution/article-06.tex]: This should probably changed so it will actually compile.
```suggestion
    \item IRSeC Competition Architect
```

**Comment by Scuzz3y** [comment]
Please correct the name of the branch `CA-ammendment` to `CA-Amendment`.

**Review by s-newman — APPROVED** [review]
LGTM

**Review by Cictrone — APPROVED** [review]
remember to update the release!!!!

**Pull request merged**